.travis.yml

```diff
@@ -17,5 +15,15 @@ addons:
     packages:
       - oracle-java8-installer
       - oracle-java8-unlimited-jce-policy
+jobs:
+  include:
+    - stage: test
+      script: "./gradlew -s --scan build codeCoverageReport"
+    - stage: deploy
+      script: "./gradlew -s build && ./gradlew ciPerformRelease"
+stages:
+  - test
+  - name: deploy
```

Comment:
curious why you have name: before deploy but not test.  Are they equivalent?

Reply:
Yep, they're equivalent, but I added the name: for consistency